src/pages/SubmissionConfig.test.js

```diff
@@ -110,6 +113,16 @@ describe('SubmissionConfig workflow details', () => {
       expect(mockLeoResponse).toHaveBeenCalledTimes(0)
     })
 
+    const backButton = screen.getByText('Back to workflows')
+
+    // ** ACT **
+    // user clicks on back button
+    await act(async () => {
+      await userEvent.click(backButton)
```

Comment:
TOL: this can be a separate small unit test since it is unrelated to the test it is currently in